.github/workflows/nightly-scan.yml

```diff
@@ -30,13 +30,15 @@ jobs:
       - name: Set job environment variables
         run:  cat JOB.env >> $GITHUB_ENV
       - name: Scan Java docker image
-        run: |
-          curl -s https://ci-tools.anchore.io/inline_scan-v$ANCHORE_SCAN_VERSION | \
-          bash -s -- -r -f -p -b ./anchore-policy.json defradigital/$IMAGE_NAME:$DEFRA_VERSION-java${{matrix.image.runTimeVersion}}
-      
-      - name: Archive anchore report
+        id: scan
+        uses: anchore/scan-action@v3
+        with:
+          image: defradigital/$IMAGE_NAME:$DEFRA_VERSION-java${{matrix.image.runTimeVersion}}
```

Comment:
I ran into an issue with the command line style env var with the nightly scan and had to replace them with
`${{env.IMAGE_NAME}}:${{env.DEFRA_VERSION}}` to get it working